### DISCUSSION TITLE
New version starts laser in centre?!

### DISCUSSION THREAD

**Discussion by hucker75** [arkypita/LaserGRBL#2277] [Q&A]
In the previous version, it was set to start at the bottom left, which is obviously where my laser is physically to start with.  I can't possibly align it dead centre without the edge to go by.  I now have no idea how to use the software.  Thanks a lot!

**Comment by arkypita** [comment]
Nothing has changed in the new version about how the laser work or where the laser start. Please check your own configuration.

**Comment by hucker75** [comment]
I worked it out eventually.  The interface changed drastically, a dialog box explaining how to use it would have been nice.  I thought it was expecting the laser to start in the centre, when all that happened was for some reason the screen starts with 0 in the centre.  Don't all lasers start bottom left so you can get the workpiece under?
